src/lib/wizard/components/Deploy.svelte

```diff
@@ -415,7 +417,9 @@
     <div class="flex flex-row items-center gap-2">
       <i class={`fa fa-lightbulb-o text-lime-600`}></i>
       <div class="text-xs text-gray-600">
-        <p>Ensure you have an Explorer API Key set in your <u><a href='{deploymentEnvironmentUrlWithFallback}' target='_blank'>Deploy Environment</a></u> for the current network to allow the contract to be verified automatically.</p>
+        {#if !hasSetUpBlockExplorerKeyForCurrentNetwork}
```

Comment:
@ericglau does this align with what you had in mind for block explorer API key hint?

Reply:
Commented in the lines below